docker-compose.yml

```diff
@@ -45,13 +48,21 @@ services:
       dockerfile: ./docker/Dockerfile.webrequest
     shm_size: 2gb
     volumes:
+      - ./.env:/app/webrequest/.env
       - ./webrequest:/app/webrequest
+      # below should be not needed, but copy doesn't work
+      - ./docker:/app/docker
```

Comment:
this is because the docker folder is `.dockerignore`d. If you move `webrequest.sh` to `webrequest` folder it should work

Reply:
removed in 9539ce042d8c775ca019017333961e215ead1dfb